.github/workflows/sandbox-deletion.yml

```
@@ -0,0 +1,41 @@
+name: Trigger Sandbox Deletion
+
+on:
```

yamllint (1.29.0-1)

[warning] truthy value should be one of [false, true]

(truthy)